app/src/org/commcare/fragments/connect/ConnectDeliveryProgressDeliveryFragment.java

```diff
@@ -136,6 +139,10 @@ public void updateView() {
         TextView textView = view.findViewById(R.id.connect_progress_progress_text);
         textView.setText(String.format(Locale.getDefault(), "%d%%", percent));
 
+        if(job.getIsUserSuspended()){
+            button.setEnabled(false);
```

Comment:
Just to ensure the correct state (regardless of XML config), how about the following instead of the 'if' structure:
button.setEnabled(!job.getIsUserSuspended());